nixnet/session.py

```diff
@@ -25,6 +25,35 @@
 
 
 class Session(object):
+    """What is a Session?
+
+    The NI-XNET session represents a connection between your National Instruments
+    CAN/FlexRay/LIN hardware and hardware products on the external network.
+
+    Each session configuration includes:
+        Interface: This specifies the National Instruments hardware to use.
+        Database objects: These describe how external hardware communicates.
+        Mode: This specifies the direction and representation of I/O data.
+
```

Comment:
A session doesn't contain these but is initialized with them.  This belongs in the `__init__` docstring